packages/event-storage-adapter-http/src/adapter.ts

```diff
@@ -0,0 +1,82 @@
+import uniqBy from 'lodash/uniqBy';
+import { OpenAPI } from 'openapi-types';
```

Comment:
Tu peux utiliser des `import type {...}` si jamais ce n'est que des types ?

Reply:
c'est un [namespace](https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html#using-namespaces) et pas un type du coup ça s'importe comme un module